docs/developers/contributing-guidelines.md

```diff
@@ -0,0 +1,142 @@
+# Code contributions
+
+Project and code leads are experienced HOT volunteer and staff developers and the main points of contact for the project. They are also the final reviewers of issues and pull requests. Code leads will review pull requests and provide feedback. The purpose of this role is to help contributors,
+provide consistency and ensure code quality.
+
+All of the development is going to happen in the [project repository](https://github.com/hotosm/tasking-manager) and everything we work on shall be related to and documented in issues of the related [issue queue](https://github.com/hotosm/tasking-manager/issues).
+
+## Code basics
+
+1. Write tests for all new backend features and use a tool (like coveralls.io) to measure test coverage.
+2. Consider writing tests when building new frontend.
+3. Stick to pep8 python style guide for the backend.
+4. Apply ESLint and [prettier](https://prettier.io/) style guide rules for the frontend code.
+5. Export translatable strings with `make refresh-translatables` and include them in your commit.
+
+## Code collaboration and version control
+
+*Note: We use **git flow** as our branching model. Read more [here](https://nvie.com/posts/a-successful-git-branching-model/) and [here](https://jeffkreeftmeijer.com/git-flow/), and refer to this [cheatsheet](https://danielkummer.github.io/git-flow-cheatsheet/) if you aren’t familiar with it yet. You probably want to install a [helper application](https://github.com/nvie/gitflow/wiki/Installation) to facilitate the flow a bit.*
+
+* Use branches in the tasking-manager project. This allows others to rebase your branch when they are reviewing or to continue started work. We follow git flow’s naming convention
+  - `feature/ISSUENUMBER-SHORT-TITLE-SEPARATED-BY-HYPHENS` for general new features you are working on
+  - `hotfix/ISSUENUMBER-SHORT-TITLE-SEPARATED-BY-HYPHENS` for important bug fixes that need to go into the main releases as soon as possible
```

Comment:
How about adding a new branch `coldfix` or `bugfix` which fixes bugs but isn't that urgent to be deployed to main release as soon as possible.